src/REPLMode.jl

```diff
@@ -1203,6 +1238,41 @@ is modified.
     """,
 ),
 ], #package
+
+["registry"] => CommandDeclaration[
+(
+    CMD_REGISTRY_ADD,
+    ["add"],
+    do_registry_add!,
+    (ARG_RAW, []),
```

Comment:
@00vareladavid I could use your help here; currently I put `ARG_RAW` here, and manually call `parse_registry` in the `do_` functions, but I guess I should instead have something like `ARG_REGISTRY` and handle that earlier in the pipeline, such that the `do_` functions obtain `RegistrySpec`s instead of the raw argument? Not sure how to do this, help appreciated.

Reply:
Sure! I mentioned [here](https://github.com/JuliaLang/Pkg.jl/pull/585#issuecomment-410911201) that this part of the spec needs more work. I was planning on including including it with some other changes, but I'll get it done first instead. It should support easily adding new tokens.

Reply:
Hopefully #595 will do it. `CMD_REGISTRY_ADD` would use `(ARG_RAW, [])` -> `([], parse_registry, [:add => true])`. The other ones would be  `([], parse_registry, [])` I think